.eslintrc.yaml

```diff
@@ -3,20 +3,21 @@ env:
   node: true
   jest: true,
 extends:
-  - "plugin:@typescript-eslint/eslint-recommended"
-  - "plugin:@typescript-eslint/recommended"
-  - "prettier"
-  - "prettier/@typescript-eslint"
-parser: "@typescript-eslint/parser"
+  - 'plugin:@typescript-eslint/eslint-recommended'
+  - 'plugin:@typescript-eslint/recommended'
+  - 'prettier'
+  - 'prettier/@typescript-eslint'
+parser: '@typescript-eslint/parser'
 rules:
-  "@typescript-eslint/no-parameter-properties": "off"
-  "@typescript-eslint/interface-name-prefix": "off"
-  "@typescript-eslint/explicit-function-return-type": "off"
-  "@typescript-eslint/no-explicit-any": "off"
-  "@typescript-eslint/no-empty-function": "off"
-  "@typescript-eslint/no-unused-vars": "off"
-  "unused-imports/no-unused-imports-ts": 2
-  "unused-imports/no-unused-vars-ts": 1
-plugins: 
-  - "@typescript-eslint/eslint-plugin"
-  - "unused-imports"
\ No newline at end of file
+  '@typescript-eslint/no-parameter-properties': 'off'
+  '@typescript-eslint/interface-name-prefix': 'off'
+  '@typescript-eslint/explicit-function-return-type': 'off'
+  '@typescript-eslint/no-explicit-any': 'off'
+  '@typescript-eslint/no-empty-function': 'off'
+  '@typescript-eslint/no-unused-vars': 'off'
+  '@typescript-eslint/explicit-module-boundary-types': 'off'
```

Comment:
I feel like we should have this turned on https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

> Explicit types for function return values and arguments makes it clear to any calling code what is the module boundary's input and output.
>Consider using this rule in place of no-untyped-public-signature which has been deprecated.

I know this is the source of the eslint lint warnings (which should be addresses).

As this is an SDK, we should enforce that all exported methods have a defined return type, using actual types and not `any`.